app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use App\Contracts\LoginContract;
+use App\User;
+
+class LoginController extends Controller
```

Comment:
May want to rename to include the idea that logout is in here, couldnt tell by just the name.